tests/e2e/configurer/current.go

```diff
@@ -44,7 +45,16 @@ func (cb *CurrentBranchConfigurer) ConfigureChain(chainConfig *chain.Config) err
 		return err
 	}
 	cb.t.Logf("temp directory for chain-id %v: %v", chainConfig.Id, tmpDir)
-	initializedChain, err := initialization.InitChain(chainConfig.Id, tmpDir, chainConfig.ValidatorInitConfigs, time.Duration(chainConfig.VotingPeriod), 0)
+
+	numVal := float32(len(chainConfig.ValidatorInitConfigs))
+
+	chainConfig.VotingPeriod = config.PropDepositBlocks + numVal*config.PropVoteBlocks + config.PropBufferBlocks
+	chainConfig.ExpeditedVotingPeriod = config.PropDepositBlocks + numVal*config.PropVoteBlocks + config.PropBufferBlocks - 1
+
```

Comment:
Seems that we do a similar calculation in `upgrade.go`. Can we encapsulate this into a shared function or a method defined on the chain config?